actionview/lib/action_view/template/resolver.rb

```diff
@@ -178,36 +178,51 @@ def clear_cache
       super()
     end
 
+    def find_all(name, prefix = nil, partial = false, details = {}, key = nil, locals = [])
+      locals = locals.map(&:to_s).sort!.freeze
+
+      cached(key, [name, prefix, partial], details, locals) do
+        find_templates(name, prefix, partial, details, locals, cache: !!key)
```

Comment:
To avoid this duplication maybe we should define a new private method that is called on `Resolver#find_all` and override only that method?

Reply:
I've tried to do this